test/common/fixtures.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable required-modules */
+'use strict';
+
+const path = require('path');
+const fs = require('fs');
+
+const fixturesDir = path.join(__dirname, '..', 'fixtures');
+
+function fixturesPath(...args) {
+  return path.join(fixturesDir, ...args);
+}
+
+function readFixtureSync(args, enc) {
+  if (Array.isArray(args))
+    return fs.readFileSync(fixturesPath(...args), enc);
+  return fs.readFileSync(fixturesPath(args), enc);
+}
+
+function readFixtureKey(name, enc) {
+  return fs.readFileSync(fixturesPath('keys', name), enc);
```

Comment:
Take another look at the signatures. `readFixtureSync(...args)` takes an arbitrary number of args that are concatenated using `path.join(...args)`, whereas `readFixtureKey(name, enc)` takes a single path segment and an optional encoding. Yes, `readFixtureKey()` uses `readFixtureSync()` but they are not equivalent to one.